src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -4,7 +4,6 @@
 using System;
```

Comment:
@daxian-dbw, your last commit had 4 failures in `PowerShell-CI-windows`
Basic EventLog tests on Windows.Verifies scriptblock logging: normal script block
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Logging.Tests.ps1: line 488
488:         $created | Should -Not -BeNullOrEmpty
```
Basic EventLog tests on Windows.Verifies scriptblock logging: script block with Null
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Logging.Tests.ps1: line 488
488:         $created | Should -Not -BeNullOrEmpty
```
ConsoleHost unit tests.-SettingsFile Commandline switch.Verifies PowerShell writes to the custom -settingsFile
```powershell
Expected strings to be the same, but they were different.
Expected length: 9
Actual length:   12
Strings differ at index 0.
Expected: 'AllSigned'
But was:  'RemoteSigned'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 357
357:             $content.'Microsoft.PowerShell:ExecutionPolicy' | Should -Be $expectedValue
```
ConsoleHost unit tests.-SettingsFile Commandline switch.Verify PowerShell removes a setting from the custom -settingsFile
```powershell
Expected $null, but got 'RemoteSigned'.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 370
370:             $content.'Microsoft.PowerShell:ExecutionPolicy' | Should -Be $null
```